src/utils/fetchClient.ts

```diff
@@ -0,0 +1,47 @@
+import { ErrorMessages } from '../types/ErrorNessages';
+
+const BASE_URL = 'https://mate.academy/students-api';
+
+// returns a promise resolved after a given delay
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)